docs/cli/execution_command.md

```
@@ -40,25 +40,9 @@ Type `!` on its own to list the available tools and plugins.
-  - name: dags                      # a command group with nested subcommands
-    description: DAG operations
-    subcommands:
-      - name: trigger
-        args:
-          - {name: dag_id, required: true}
-          - {name: --conf, description: run configuration JSON}
-      - name: list
-```
+- When a plugin provides command metadata, the `!<plugin>` completer lists its
+  commands and hints their arguments. Completion follows nested command groups,
+  so typing `!airflow dags ` can offer commands such as `list` and `trigger`.
```

markdownlint-cli2 (0.23.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)